test/extensions/staked-expenditure.js

```diff
@@ -330,6 +330,26 @@ contract("StakedExpenditure", (accounts) => {
       );
     });
 
+    it("cannot slash the stake if the expenditure is already cancelled", async () => {
```

Comment:
Are these changes related to the rest of the PR, or did you just notice they were missing as you went along?

Reply:
I assume I found them as low-hanging fruit when trying to get coverage up.